timestamp-task/src/main/java/io/spring/timestamp/TimestampTaskProperties.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2021 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.spring.timestamp;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.util.Assert;
+
+/**
+ * @author Glenn Renfro
+ */
+@ConfigurationProperties
```

Comment:
@cppwfs before this was prefixed at `timestamp` and now its ``. I would favor using a prefix as it's a best practice (and ppl will copy/pasta this I am sure). Wdyt?